### ISSUE TITLE
Fix build with expo

### ISSUE THREAD

**Issue by fabien0102** [fabien0102/climbing-app#1]
Strange issue with expo build and graphql files

```
Unable to resolve module `./createUserMutation.graphql` from `D:\Projets\climbing-app\app\queries\withCreateUser.js`: could not resolve `D:\Projets\climbing-app\app\queries\createUserMutation.graphql' as a file nor as a folder
```

**Comment by fabien0102** [comment]
Closed by #2

**Comment by fabien0102** [comment]
![image](https://user-images.githubusercontent.com/1761469/31358154-c37bc552-ad44-11e7-8664-dbe8f15395db.png)

**Comment by fabien0102** [comment]
closed #4
